src/components/editor/EditorAppController.jsx

```diff
@@ -120,7 +120,7 @@ export default class EditorAppController extends BaseComponent {
   }
 
   render() {
-    const navItems = ['Articles', 'Authors', 'Issues', 'Images'];
```

Comment:
It seems like this is outdated since the prop in `EditorNavigation` is never used, so let's just completely remove this, and the prop that is passed into `EditorNavigation` yeah? (not your fault, was already there)

Reply:
Yeah so if you could remove this last thing as well I'll merge it right in :)

Reply:
Do you mean removing the whole line, or just the Readme from it? If I remove the line the page breaks.

Reply:
Yeah but it shouldn't if you also remove the prop that's passed into `EditorNavigation` as I mentioned above, right? As I can see in the `EditorNavigation` source that we don't use that prop

Reply:
I think it's an artifact from when Zane redid the UI of the admin UI. Probably he used it for some time and then changed it and forgot that part.

Reply:
oh gotcha! it should work now.